### DISCUSSION TITLE
a way to use/map 'hashtag' - key?

### DISCUSSION THREAD

**Discussion by workingj** [helix-editor/helix#6097] [Q&A] [answered]
hi, is there a way to use `#` - key?
i like to have something like `C-hash = "toggle-comments"`
have not found a way to accomplish this...
or is the usage of that key not yet implemented ?

**Comment by the-mikedavis** [comment, marked as answer]
You can use the `#` character literally but it needs to be quoted in order to not be interpreted as a comment. You should be able to bind Ctrl and `#` with `"C-#"` but it looks to me like that key combination is not passed from the terminal emulator to the app running inside, so it isn't possible to bind that combination.

**Reply by workingj** [reply]
thx @the-mikedavis, same here, the key isn't passed through...
